### ISSUE TITLE
Quartz test failed with java8

### ISSUE THREAD

**Issue by zemian** [quartz-scheduler/quartz#60]
Setup:
```
Zemians-Air:quartz-test zemian$ mvn -version
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T11:41:47-05:00)
Maven home: /Users/zemian/apps/apache-maven-3.3.9
Java version: 1.8.0_60, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_60.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.11.6", arch: "x86_64", family: "mac"
```

Output:
```
Results :

Failed tests:   testFired(org.terracotta.quartz.upgradability.serialization.CalendarIntervalTriggerImplSerializationTest): Unrecognized serialized form saved to /Users/zemian/javadev/zemian/quartz-test/quartz/CalendarIntervalTriggerImpl-2.ser
  testConstructed(org.terracotta.quartz.upgradability.serialization.CalendarIntervalTriggerImplSerializationTest): Unrecognized serialized form saved to /Users/zemian/javadev/zemian/quartz-test/quartz/CalendarIntervalTriggerImpl-3.ser
  testFired(org.terracotta.quartz.upgradability.serialization.CronTriggerImplSerializationTest): Unrecognized serialized form saved to /Users/zemian/javadev/zemian/quartz-test/quartz/CronTriggerImpl-2.ser
  testConstructed(org.terracotta.quartz.upgradability.serialization.CronTriggerImplSerializationTest): Unrecognized serialized form saved to /Users/zemian/javadev/zemian/quartz-test/quartz/CronTriggerImpl-3.ser
  testFired(org.terracotta.quartz.upgradability.serialization.DailyTimeIntervalTriggerImplSerializationTest): Unrecognized serialized form saved to /Users/zemian/javadev/zemian/quartz-test/quartz/DailyTimeIntervalTriggerImpl-2.ser
  testConstructed(org.terracotta.quartz.upgradability.serialization.DailyTimeIntervalTriggerImplSerializationTest): Unrecognized serialized form saved to /Users/zemian/javadev/zemian/quartz-test/quartz/DailyTimeIntervalTriggerImpl-3.ser
  testEmptyAllowTransientsMap(org.terracotta.quartz.upgradability.serialization.JobDataMapSerializationTest): Unrecognized serialized form saved to /Users/zemian/javadev/zemian/quartz-test/quartz/JobDataMap-2.ser
  testEmptyMap(org.terracotta.quartz.upgradability.serialization.JobDataMapSerializationTest): Unrecognized serialized form saved to /Users/zemian/javadev/zemian/quartz-test/quartz/JobDataMap-3.ser
  testFired(org.terracotta.quartz.upgradability.serialization.SimpleTriggerImplSerializationTest): Unrecognized serialized form saved to /Users/zemian/javadev/zemian/quartz-test/quartz/SimpleTriggerImpl-2.ser
  testConstructed(org.terracotta.quartz.upgradability.serialization.SimpleTriggerImplSerializationTest): Unrecognized serialized form saved to /Users/zemian/javadev/zemian/quartz-test/quartz/SimpleTriggerImpl-3.ser

Tests run: 51, Failures: 10, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] quartz-checkstyle .................................. SUCCESS [  1.336 s]
[INFO] quartz-parent ...................................... SUCCESS [  0.076 s]
[INFO] quartz-core ........................................ SUCCESS [05:49 min]
[INFO] quartz-commonj ..................................... SUCCESS [  1.114 s]
[INFO] quartz-jboss ....................................... SUCCESS [  1.384 s]
[INFO] quartz-stubs ....................................... SUCCESS [  0.886 s]
[INFO] quartz-oracle ...................................... SUCCESS [  1.091 s]
[INFO] quartz-weblogic .................................... SUCCESS [  0.904 s]
[INFO] quartz-jobs ........................................ SUCCESS [  3.985 s]
[INFO] quartz-plugins ..................................... SUCCESS [  2.853 s]
[INFO] quartz-terracotta-root ............................. SUCCESS [  0.002 s]
[INFO] quartz-terracotta-bootstrap ........................ SUCCESS [  1.731 s]
[INFO] quartz ............................................. FAILURE [  4.666 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 06:09 min
[INFO] Finished at: 2016-10-11T22:35:21-04:00
[INFO] Final Memory: 21M/212M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.12.4:test (default-test) on project quartz: There are test failures.
[ERROR] 
[ERROR] Please refer to /Users/zemian/javadev/zemian/quartz-test/quartz/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :quartz
```

**Comment by zemian** [comment]
Using JDK6 + MVN 3.2 works obviously.
```
Zemians-Air:quartz-test zemian$ mvn -version
Apache Maven 3.2.5 (12a6b3acb947671f09b81f49094c53f426d8cea1; 2014-12-14T12:29:23-05:00)
Maven home: /Users/zemian/apps/apache-maven-3.2.5
Java version: 1.6.0_65, vendor: Apple Inc.
Java home: /Library/Java/JavaVirtualMachines/1.6.0.jdk/Contents/Home
Default locale: en_US, platform encoding: MacRoman
OS name: "mac os x", version: "10.11.6", arch: "x86_64", family: "mac"
```

**Comment by zemian** [comment]
Using JDK7 + MVN 3.3 also works fine.

```
mvnw.bat -version
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T11:41:47-05:00)
Maven home: C:\Users\zedeng\.m2\wrapper\dists\apache-maven-3.3.9-bin\2609u9g41na2l7ogackmif6fj2\apache-maven-3.3.9
Java version: 1.7.0_80, vendor: Oracle Corporation
Java home: D:\apps\jdk1.7.0_80\jre
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 7", version: "6.1", arch: "amd64", family: "windows"
```

**Comment by vpavic** [comment]
Build using JDK7 actually fails for me:

```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] quartz-checkstyle .................................. SUCCESS [  1.197 s]
[INFO] quartz-parent ...................................... SUCCESS [  0.569 s]
[INFO] quartz-core ........................................ SUCCESS [05:29 min]
[INFO] quartz-commonj ..................................... SUCCESS [  0.913 s]
[INFO] quartz-jboss ....................................... SUCCESS [  0.930 s]
[INFO] quartz-stubs ....................................... SUCCESS [  0.735 s]
[INFO] quartz-oracle ...................................... SUCCESS [  0.921 s]
[INFO] quartz-weblogic .................................... SUCCESS [  0.856 s]
[INFO] quartz-jobs ........................................ FAILURE [  6.109 s]
[INFO] quartz-plugins ..................................... SKIPPED
[INFO] quartz-terracotta-root ............................. SKIPPED
[INFO] quartz-terracotta-bootstrap ........................ SKIPPED
[INFO] quartz ............................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 05:41 min
[INFO] Finished at: 2016-10-15T22:42:13+02:00
[INFO] Final Memory: 31M/410M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9:jar (create-javadoc) on project quartz-jobs: MavenReportException: Error while creating archive:
[ERROR] Exit code: 1 - javadoc: error - invalid flag: -Xdoclint:none
[ERROR] 
[ERROR] Command line was: /usr/lib/jvm/java-7-oracle/jre/../bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/home/vedran/dev/projects/quartz/quartz-jobs/target/apidocs' dir.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :quartz-jobs
```

```
$ ./mvnw -v
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T17:41:47+01:00)
Maven home: /home/vedran/.m2/wrapper/dists/apache-maven-3.3.9-bin/2609u9g41na2l7ogackmif6fj2/apache-maven-3.3.9
Java version: 1.7.0_80, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-7-oracle/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.8.0-22-generic", arch: "amd64", family: "unix"
```

The cause appears to be the change made by #50.

**Comment by zemian** [comment]
Yeah, it's failing on javadoc processing. I already raise issues #63 for this regression problem. This is different than the original problem with JDK8 which actually resulted by a failed test.

**Comment by zemian** [comment]
Fixed in PR #67
